src/rrt-viewer/RRTWidget.cpp

```diff
@@ -13,7 +13,7 @@ const double VelocityDrawingMultiplier = 12;
 RRTWidget::RRTWidget() {
     Vector2d size(800, 600);
     _stateSpace = make_shared<GridStateSpace>(size.x(), size.y(), 40, 30);
-    _biRRT = new BiRRT<Vector2d>(_stateSpace);
+    _biRRT = new BiRRT<Vector2d>(_stateSpace, RRT::hash, dimensions);
```

Comment:
If you feel up to it, it would be sort of nice to get rid of this memory leak (that only happens right now when we close the program so it doesn't really matter at all), by changing _biRRT to a unique_ptr.